config/combined-worker-kafka-compose.yaml

```diff
@@ -0,0 +1,72 @@
+version: '2'
```

Comment:
I've got some ideas for improvements we can make with this file but they can wait for another day.